### ISSUE TITLE
Some tables are not created with bigint type during fresh installation of Nextcloud 15

### ISSUE THREAD

**Issue by PanCakeConnaisseur** [nextcloud/server#13080]
### Steps to reproduce
1. Install Nextcloud from https://download.nextcloud.com/server/releases/latest-15.tar.bz2
2. 
```
php occ maintenance:install --database "pgsql" --database-name "nextcloud" --database-user "postgres" --database-pass "USE_PASSWORD_DB_POSTGRES" --admin-user "admin" --admin-pass "SET_PASSWORD_NEXTCLOUD_ADMIN"
```
3. Log in as admin
4. Go to Settings -> Administration -> Overview

### Expected behaviour
No *Security & setup warnings* because its a fresh install and the install script should setup the DB with *bigint* from the start.

### Actual behaviour
*Security & setup warnings*  contains
>Some columns in the database are missing a conversion to big int. Due to the fact that changing column types on big tables could take some time they were not changed automatically. By running 'occ db:convert-filecache-bigint' those pending changes could be applied manually. This operation needs to be made while the instance is offline. For further details read the documentation page about this.

>    filecache.mtime
>    filecache.storage_mtime

### Server configuration

**Operating system**:
Debian 9

**Web server:**
Apache 2.4.25-3+deb9u6

**Database:**
PostgreSQL 9.6+181+deb9u2

**PHP version:** 7.0+49

**Nextcloud version:**
15.0.0

**Updated from an older Nextcloud/ownCloud or fresh install:**
fresh install

**Where did you install Nextcloud from:**
https://download.nextcloud.com/server/releases/latest-15.tar.bz2

### Table oc_filecache
![screenshot_20181214_195935](https://user-images.githubusercontent.com/17473656/50021958-f8fbb900-ffda-11e8-969f-38a73fb79f40.png)

**Comment by danielkesselberg** [comment]
Thank you for reporting :+1: There is already an report for this: https://github.com/nextcloud/server/issues/12927

**Comment by PanCakeConnaisseur** [comment]
I get the related error message on a fresh 17.0.2 install.

> Some columns in the database are missing a conversion to big int. Due to the fact that changing column types on big tables could take some time they were not changed automatically. By running 'occ db:convert-filecache-bigint' those pending changes could be applied manually. This operation needs to be made while the instance is offline. For further details read the documentation page about this.
> 
>mounts.storage_id
>mounts.root_id
>mounts.mount_id

The issue you linked, links to a pull request that has been merged more than a year ago, but afaics 17.0.2 came out much later. Therefore I conclude that this bug is still open and not fixed.

**Comment by kesselb** [comment]
The information is about different columns. From time to time new columns are migrated from int and bigint. So yes it's probably a reoccurring issue.

Here is the related code change: https://github.com/nextcloud/server/pull/18299 I guess it's on the roadmap to change the migration for fresh setups as well.
